src/App.css

```diff
@@ -92,11 +92,11 @@ h1 + h2 {
   margin-bottom: 10px;
 }
 
-/* headding */
-h3 {
+/* heading */
```

Comment:
As a general rule. If we comment out code, we should add a comment about why it is commented out and/or why we might want to uncomment it in the future.

In this case, I think we can just remove this code.

Reply:
For this one, have removed the commented code.